py3status/modules/scratchpad_async.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display the amount of windows and indicate urgency hints on scratchpad (async).
+Display total number of windows and indicate urgency hints on your i3 scratchpad asynchronously.
```

Comment:
Display number of windows and indicate urgency hints on scratchpad asynchronously. (?) ... No urgency hints on `scratchpad_counter`.